### PR TITLE
add stage1 cluster settings

### PR DESCRIPTION
**Pull request by umezawatakeshi**
This PR should be merged after https://github.com/cybozu-go/neco/issues/2179 is RELEASED to stage0.

- build releases for `test-*` tags too
- add stage1 cluster settings
- add `dev` cluster type, which tracks `test-*` tag.

Signed-off-by: UMEZAWA Takeshi <takeshi-umezawa@cybozu.co.jp>

### CODE REVIEW

**Review by kmdkuk — DISMISSED** [review]
LGTM

**Review by masa213f — CHANGES_REQUESTED** [review]

File: worker/operator.go [hunk 1]
```diff
@@ -115,6 +115,9 @@ func (o *operator) UpdateNeco(ctx context.Context, req *neco.UpdateRequest) erro
 	if env == neco.TestEnv {
 		return installLocalPackage(ctx, deb)
 	}
+	if env == neco.TestEnv {
```
Comment (masa213f) [inline worker/operator.go:118]: Like this?
```suggestion
	if env == neco.DevEnv {
```

**Review by masa213f — CHANGES_REQUESTED** [review]

**Review by masa213f — APPROVED** [review]
LGTM

**Comment by umezawatakeshi** [comment]
rebased (due to artifacts.go)

**Review by masa213f — APPROVED** [review]
LGTM

**Review by kmdkuk — APPROVED** [review]
LGTM

**Pull request merged**